src/blocks/profile.css

```diff
@@ -8,9 +8,48 @@
   color: white;
 }
 
-.profile__avatar {
+.profile__avatar-wrapper {
   margin-right: 30px;
+  max-width: 120px;
+  max-height: 120px;
+  background-color: transparent;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  cursor: pointer;
+}
+
+.profile__avatar-wrapper:hover {
+  opacity: 0.5;
+}
+.edit-avatar-button {
+  z-index: 1;
+}
+.profile__avatar-wrapper:hover .edit-avatar-button {
+  z-index: 2;
+}
+
+.profile__avatar {
   border-radius: 100%;
+  z-index: 1;
+  max-width: 120px;
+  max-height: 120px;
+}
+
+.avatar-edit-button {
+  position: inherit;
+  width: 26px;
+  height: 26px;
+  border: 0;
+  cursor: pointer;
+  position: relative;
+  left: 250%;
+  opacity: 1;
+}
+
+.edit-avatar-button:hover {
+  opacity: 0.7;
 }
 
```

Comment:
* See here http://joxi.ru/eAOOB91CGdod8A When I delete 3 cards in a row there are errors
* Now all your cards (which you've tried to delete but changed your mind) are **deleted** all at once. Please try it yourself: add 3 cards in a row then try to delete them but don't do it (cancel and close the confirmation popup) and then delete any other card - and all your previous cards are deleted at once (all 3 and the other one). It’s because of a **memory leak**. All listeners are gathered in one element and they are fired all together when you confirm deleting. You need either to **correctly remove** submit listeners or update the card’s data. Please, consult your mentor about that.